creator/lib/firebase/database.dart

```diff
@@ -2,7 +2,11 @@ import 'package:cloud_firestore/cloud_firestore.dart';
 import 'package:creator/models/user_model.dart';
 import 'package:firebase_auth/firebase_auth.dart';
 
-class DbHandler {
+abstract class DbHandlerAbstract {
```

Comment:
Why use abstract class?

Reply:
There hava a error but this ticket cannot build app.
so I'll fix as other issue.

`'updateUser' must have a method body because 'DbHandlerAbstract' isn't abstract.
Try making 'DbHandlerAbstract' abstract, or adding a body to 'updateUser'.dart[concrete_class_with_abstract_member](https://dart.dev/diagnostics/concrete_class_with_abstract_member)`

Reply:
https://github.com/KingdomSeekersJP/salon_app/issues/20